cmd/example/main.go

```diff
@@ -54,7 +55,19 @@ func main() {
 	defer func() {
 		_ = closeF(context.Background())
 	}()
-	result, err := cl.GetString(ctx, config)
+	var result any
+	switch cfgType {
+	case "string":
+		result, err = cl.GetString(ctx, config)
+	case "int":
+		result, err = cl.GetInt(ctx, config)
+	case "float":
+		result, err = cl.GetFloat(ctx, config)
+	case "bool":
+		result, err = cl.GetBool(ctx, config)
+	default:
```

Comment:
not supporting json/proto for now as that requires initializing `result` before calling the sdk. will let users do that explicitly if they need to test.